nova/core/roles/accounts/defaults/main.yml

```diff
@@ -6,6 +6,8 @@ domain_user_accounts: []
 domain_groups: []
 sudo_requires_password: true
 use_ssh_agent_for_sudo: false
+accounts_remove_stale: true # Compares the accounts list with the users on the system createrd by this role and removes any that are not in the accounts list
+fresh_deploy: false # Checks if the machine has just been created, required for setting correct connection paramaters for AWS machines
```

Comment:
This var is not used in the account role. I believe that it should be removed or relocated to AWS connection.

Reply:
It is used in `- name: Setting {{ ansible_deployer_username }} as connection user...` task. If you remember the original idea was that all roles should be usable independently so all variables that are used in the role should also have a default value.

Reply:
Indeed, I didn't find the task using it in account role :)